sci-biology/fsl/files/fsl-5.0.9-fsldir_redux.patch

```diff
@@ -0,0 +1,452 @@
+diff -Naur fsl.orig/src/fast4/fast_two.cc fsl/src/fast4/fast_two.cc

```

Comment:
Thanks, could you please include patch description to these files?

Preferably according to https://dev.gentoo.org/~vapier/clean-patches and https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html